### PR TITLE
CLC-6161, minor cleanup in footer; remove /tools ref

### PR DESCRIPTION
**Pull request by johncrossman**
https://jira.ets.berkeley.edu/jira/browse/CLC-6161

Clean up footer. Changes in `server_runtime.rb` are strictly code style.

### CODE REVIEW

File: src/assets/templates/footer.html [hunk 1]
```diff
@@ -37,7 +37,7 @@
       <li>Current DB Schema: <span data-ng-bind="versions.currentDbSchema"></span></li>
       <li>Previous Release DB Schema: <span data-ng-bind="versions.previousReleaseDbSchema"></span></li>
       <li>Up Since: <span data-ng-bind="firstVisited"></span></li>
-      <li>Links: <a data-ng-href="https://github.com/ets-berkeley-edu/calcentral/commit/{{gitCommit}}">Git commit</a>, <a href="/tools/styles">Styles Reference</a></li>
+      <li>Git commit: <a data-ng-href="https://github.com/ets-berkeley-edu/calcentral/commit/{{gitCommit}}">{{gitCommit | limitTo: 7}}</a></li>
```
Comment (christianvuerings) [inline src/assets/templates/footer.html]: @johncrossman would it be possible to use `data-ng-bind` instead of `{{}}` inside the `<a>` tag?

Comment (johncrossman) [inline src/assets/templates/footer.html]: @christianvuerings I had no luck finding code sample with data-ng-bind + limitTo. Your recommendation?

Comment (christianvuerings) [inline src/assets/templates/footer.html]: @johncrossman so `<a data-ng-href="https://github.com/ets-berkeley-edu/calcentral/commit/{{gitCommit}}" data-ng-bind="gitCommit | limitTo: 7"></a>` doesn't work for you?

https://jsbin.com/teqobi/3/edit?html,css,js,output

**Comment by johncrossman** [comment]
Good tip, @christianvuerings  Commit has been amended.

**Pull request merged**